website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -28,6 +28,10 @@ If it’s your first time working with OpenBCI and you own the Cyton or Cyton+Da
 
 _Note: The [CSR Dongle](https://github.com/openbci-archive/Docs/blob/master/Deprecated%20Docs/Setup_CSR_Dongle.md) is no longer supported._ We recommend using the [newer Ganglion Dongle](https://shop.openbci.com/products/ganglion-dongle).
 
+:::important MacOS BLE Workaround
+If you are running MacOS 11+ and having issues streaming your Ganglion, see the [MacOS BLE workaround](../../../Troubleshooting/MacOS-Ganglion-BLE-Workaround).
```

Comment:
Awesome! This looks gross with all the ../ but the CI/CD will check that this link is valid.

Reply:
This is why we use relative links in this repo.